lib/block-supports/border.php

```diff
@@ -17,7 +17,7 @@ function gutenberg_register_border_support( $block_type ) {
 		$block_type->attributes = array();
 	}
 
-	if ( block_has_support( $block_type, array( '__experimentalBorder' ) ) && ! array_key_exists( 'style', $block_type->attributes ) ) {
+	if ( block_has_support( $block_type, array( 'border', '__experimentalBorder' ) ) && ! array_key_exists( 'style', $block_type->attributes ) ) {
```

Comment:
I haven't tested this, but I think this needs to have two `block_has_supports` checks (one for `border` and one for `__experimentalBorder`) as an array passed to this function is treated as a path, rather than an OR. I see there's a similar check down in line 158 that looks correct to me.

Reply:
Thanks for your suggestions @andrewserong 

I have updated this case in the recent commit.